src/main/java/it/gov/pagopa/service/PortalConsentService.java

```diff
@@ -1,7 +1,12 @@
 package it.gov.pagopa.service;
 
+import it.gov.pagopa.dto.PortalConsentDTO;
 import org.springframework.stereotype.Service;
 
 @Service
```

Comment:
an Interface should not be annotated as a Spring Bean, you should annotate only classes with this type of annotation